src/plugins/auto_batch/tests/unit/async_infer_request_test.cpp

```diff
@@ -122,9 +122,10 @@ class AutoBatchAsyncInferRequestTest : public ::testing::TestWithParam<AutoBatch
         auto reshaped = m_model->clone();
         auto inputs = reshaped->inputs();
         std::map<ov::Output<ov::Node>, ov::PartialShape> partial_shapes;
```

Comment:
```suggestion
        std::map<std::size_t, ov::PartialShape> partial_shapes;
```
It will be more robust, because port comparison will be avoided.